tests/test_forcing.py

```diff
@@ -35,3 +35,14 @@ def test_precip():
     # Testing with freq argument
     pout_freq = precip(p_precip, grid, freq="H")
     assert pout_freq.sizes["time"] == 313
+
+
+def test_pet():
+    cat = DataCatalog()
+    et_data = cat.get_rasterdataset("era5_daily_zarr")
+    dem = cat.get_rasterdataset("era5_orography").squeeze("time").drop("time")
+
+    peto = pet(et_data, et_data, dem, method="penman-monteith_tdew")
+
+    assert peto.raster.shape == dem.raster.shape
+    np.testing.assert_almost_equal(peto.mean(), 0.57746, decimal=4)
```

Comment:
when I run the test I get the following value: 0.57739717
still very close to what you have but not exactly the same.

Reply:
??? no clue